enhancements/20220720-alertmanager.md

```diff
@@ -0,0 +1,303 @@
+# Harvester supports `Alertmanager`
+
+Customers wish to enable Prometheus `Alertmanager` in Harvester with the ability to configure in UI and API interfaces.
+
+## Summary
+.
+
+### Related Issues
+
+https://github.com/harvester/harvester/issues/2517
+
+## Motivation
+
+`Alert` is the very important information for monitoring one running system. Customers can integrate it with other systems to have a well defined lifecycle management.
+
+### Goals
+
+Support alertmanager.
+
+Support config various types of receivers.
+
+Support customized alert rules definition.
+
+Support customized template files to format alerts
+
```

Comment:
In many deployments we upload our own alertmanager secret / deliver this via fleet / gitops process. It would be nice if we could get this posibility here, too. (example see here: https://github.com/Martin-Weiss/rancher-fleet/tree/main/rke-monitoring